documentation/source/newsfragments/1943.issue.rst

```diff
@@ -0,0 +1,11 @@
+**Cocotb must be installed through conda when using more than one conda environment**
+
+Affected users: Users of conda with more than one conda environment.
+
+Problem: When using ``pip install cocotb`` in a conda environment, binary packages are produced which are tied to this specific environment.
+After switching to another conda environment, incompatible packages are reused.
+When running simulations, users will see an error like ``libpython3.7m.so.1.0: cannot open shared object file: No such file or directory``.
+See issue :issue:`1943` for details.
+
+Solution: Conda users are encouraged to install cocotb not through pip,
+but make use of the conda package manager by running ``conda skeleton pypi cocotb; conda build cocotb`` instead.
```

Comment:
An easier solution is to just rebuild the package instead of reusing the wheel using `pip install --no-cache-dir`.